src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -109,33 +109,56 @@ typedef struct {
     /* 0x4 */ RaceWaypoint* waypoints;
 } RaceInfo; // size = 0x8
 
-static AnimationHeader* sEponaAnimations[] = {
-    &gEponaIdleAnim,
-    &gEponaWhinnyAnim,
-    &object_horse_link_child_Anim_005F64,
-    &object_horse_link_child_Anim_004DE8,
-    &gEponaWalkAnim,
-    &gEponaTrotAnim,
-    &gEponaGallopAnim,
-    &object_horse_link_child_Anim_0035B0,
-    &object_horse_link_child_Anim_003D38,
+static AnimationHeader* sEponaAnimations[ENHORSE_ANIM_MAX] = {
+    &gEponaIdleAnim,                      // ENHORSE_ANIM_IDLE
+    &gEponaWhinnyAnim,                    // ENHORSE_ANIM_WHINNY
+    &object_horse_link_child_Anim_005F64, // ENHORSE_ANIM_STOPPING
+    &object_horse_link_child_Anim_004DE8, // ENHORSE_ANIM_REARING
+    &gEponaWalkAnim,                      // ENHORSE_ANIM_WALK
+    &gEponaTrotAnim,                      // ENHORSE_ANIM_TROT
+    &gEponaGallopAnim,                    // ENHORSE_ANIM_GALLOP
+    &object_horse_link_child_Anim_0035B0, // ENHORSE_ANIM_LOW_JUMP
+    &object_horse_link_child_Anim_003D38, // ENHORSE_ANIM_HIGH_JUMP
 };
 
-static AnimationHeader* sHniAnimations[] = {
-    &gHorseIdleAnim, &gHorseShakeHeadAnim, &gHorseStopAnim,    &gHorseWhinnyAnim,   &gHorseWalkAnim,
-    &gHorseTrotAnim, &gHorseGallopAnim,    &gHorseJumpLowAnim, &gHorseJumpHighAnim,
+static AnimationHeader* sHniAnimations[ENHORSE_ANIM_MAX] = {
+    &gHorseIdleAnim,      // ENHORSE_ANIM_IDLE
+    &gHorseShakeHeadAnim, // ENHORSE_ANIM_WHINNY
+    &gHorseStopAnim,      // ENHORSE_ANIM_STOPPING
+    &gHorseWhinnyAnim,    // ENHORSE_ANIM_REARING @TODO: Does not line up
```

Comment:
Just wanted to note this. Whinny appears in 2 different entries. Horse animations are complex so I'm avoiding looking into this in great detail, just want to make a note for later and keep this PR simple